datafusion/core/src/datasource/physical_plan/parquet/mod.rs

```diff
@@ -739,7 +739,17 @@ impl ExecutionPlan for ParquetExec {
     }
 
     fn statistics(&self) -> Result<Statistics> {
-        Ok(self.projected_statistics.clone())
+        // When filters are pushed down, we have no way of knowing the exact statistics.
+        // Note that pruning predicate is also a kind of filter pushdown.
+        let stats = if self.pruning_predicate.is_some()
+            || self.page_pruning_predicate.is_some()
+            || (self.predicate.is_some() && self.pushdown_filters())
+        {
+            Statistics::new_unknown(&self.schema())
```

Comment:
Rather than discarding all statistics, what do you think about marking any precisions as `Precision::Inexact`?

I am a bit concerned that we would simply discard the statistics entirely in the (very common) case of filter pushdown 🤔

Reply:
Using `Inexact` is also acceptable. I'm just concerned that when the filter's selectivity is very low, it might lead to wasteful memory allocation.